pandas/tseries/frequencies.py

```diff
@@ -26,7 +26,7 @@
 from pandas._libs.tslibs.resolution import (Resolution,
                                             _FrequencyInferer,
                                             _TimedeltaFrequencyInferer)
-from pandas._libs.tslibs.parsing import _get_rule_month
+from pandas._libs.tslibs.parsing import _get_rule_month, _MONTH_NUMBERS
```

Comment:
same here, you can de-privatize these constants

Reply:
I think the game plan is to consolidate the several versions of these in ccalendar once it is merged.